draft-duke-quic-version-aliasing.md

```diff
@@ -104,20 +103,22 @@ entirely to the specification of the standard version.
 
 When they instantiate a connection, servers select an alternate 32-bit version
 number, and optionally an initial token extension, for the next connection at
-random and securely derive a salt and Packet Length Offset from those values
-using a repeatable process. They communicate this using a transport parameter
-extension including the version, initial token extension, salt, Packet Length
-Offset, and an expiration time for that value.
+random and securely derive a salt, packet Length Offset, and long header packet
+type codepoints from those values using a repeatable process. They communicate
+this using a transport parameter extension including the version, initial token
+extension, salt, Packet Length Offset, packet type codepoints, and an expiration
```

Comment:
"initial token extension"  do you mean the value of a token that the client includes in the Initial packet?

Reply:
yes. the server provides in the VA TP that go in the Initial Token, appended to any Retry/NEW_TOKEN that is also provided.

the purpose is to provide more than the 32 bits of entropy present in the version. It's optional but recommended.